docs/source/pages/get_started/set-up-a-project.md

```diff
@@ -337,3 +337,209 @@ that the SSH server connected to is correct (pressing `y` to proceed).
 
 Finally, your password to the central server will be requested (you will
 only need to do this once).
+
+:::
+::::
+
+(new-project-gdrive)=
+### Connecting to central storage through Google Drive
+
+Another common method of connecting to a central storage machine is via
+[Google Drive](https://drive.google.com).
+
+To set up Google Drive connection
+we need to provide:
+
+1) **gdrive_client_id:** This is the the client ID that allows you to connect to Google Drive.
+
+2) **gdrive_root_folder_id:** This is the folder ID of the root folder you want to setup
+connection to.
+
+3) **central path**: This is the path to the project *relative to the root folder*.
+
+Read about setting up your google drive client [here](https://rclone.org/drive/#making-your-own-client-id).
+
```

Comment:
I just realised that every time you change the `gdrive_root_folder_id` you need to re-run `setup_google_drive_connection`. Here we could use a [warning admonition](https://pydata-sphinx-theme.readthedocs.io/en/stable/examples/kitchen-sink/admonitions.html#caution) to inform users of this